### PR TITLE
Fasih/UPM-1128/ Remove old UI breakpoints

### PR DESCRIPTION
**Pull request by fasihali-deriv**
## Changes:

Remove old UI breakpoints and add new from deriv-com/ui

### Screenshots:

Please provide some screenshots of the change.

### CODE REVIEW

File: packages/account/src/Components/currency-selector/currency-selector.tsx [hunk 1]
```diff
@@ -92,8 +93,8 @@ const CurrencySelector = observer(
 
         const has_currency = Boolean(currency);
 
-        const { real_account_signup, real_account_signup_target, resetRealAccountSignupParams, is_desktop, is_mobile } =
-            ui;
+        const { real_account_signup, real_account_signup_target, resetRealAccountSignupParams } = ui;
+        const { isMobile, isDesktop } = useDevice();
```
Comment (amina-deriv) [inline packages/account/src/Components/currency-selector/currency-selector.tsx:98]: @fasihali-deriv THis component is part of real-account signup flow. Please check with Sergie if he is already working on this

File: packages/account/src/Components/file-uploader-container/file-uploader.tsx [hunk 2]
```diff
@@ -13,28 +13,27 @@ type TFileUploaderProps = {
...
+const UploadMessage = () => {
+    const { isMobile } = useDevice();
+
     return (
         <React.Fragment>
             <Icon icon='IcUpload' className='dc-file-dropzone__message-icon' size={30} />
             <div className='dc-file-dropzone__message-subtitle'>
                 <Text size='xxs' align='center' weight='bold' color='less-prominent'>
-                    {is_mobile ? (
+                    {isMobile ? (
```
Comment (amina-deriv) [inline packages/account/src/Components/file-uploader-container/file-uploader.tsx]: should we allow drag and drop files in tablet?

File: packages/account/src/Components/currency-selector/currency-selector.tsx (hunk 1 above)
Comment (fasihali-deriv) [reply packages/account/src/Components/currency-selector/currency-selector.tsx:98]: Oh alright. 
I thought this is related to POI. 
Thanks for highlighting

File: packages/account/src/Components/file-uploader-container/file-uploader.tsx (hunk 2 above)
Comment (fasihali-deriv) [reply packages/account/src/Components/file-uploader-container/file-uploader.tsx]: Let me check on this 😬

Comment (fasihali-deriv) [reply packages/account/src/Components/file-uploader-container/file-uploader.tsx]: I have updated the condition.

File: packages/account/src/Components/currency-selector/currency-selector.tsx [hunk 3]
```diff
@@ -266,7 +267,7 @@ const CurrencySelector = observer(
                                         )}
                                     </ThemedScrollbars>
                                 </Div100vhContainer>
-                                <Modal.Footer has_separator is_bypassed={is_mobile}>
+                                <Modal.Footer has_separator is_bypassed={isMobile}>
```
Comment (utkarsha-deriv) [inline packages/account/src/Components/currency-selector/currency-selector.tsx:270]: Hey @fasihali-deriv , may i ask how are we handling tablet view scenarios in this and following components?

File: packages/account/src/Components/file-uploader-container/file-uploader-container.tsx [hunk 4]
```diff
@@ -11,32 +11,29 @@ type TFileUploaderContainer = {
...
-            ui: { is_mobile },
-        } = useStore();
-        return (
-            <div className='file-uploader__container' data-testid='dt_file_uploader_container'>
-                {files_description}
-                {examples}
-                <Text size={is_mobile ? 'xxs' : 'xs'} as='div' className='file-uploader__file-title' weight='bold'>
-                    <Localize i18n_default_text='Upload file' />
+const FileUploaderContainer = ({ examples, files_description, onFileDrop, onError }: TFileUploaderContainer) => {
+    const { isMobile } = useDevice();
```
Comment (utkarsha-deriv) [inline packages/account/src/Components/file-uploader-container/file-uploader-container.tsx:15]: will the design for tablet and desktop be same in this component?

File: packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx [hunk 5]
```diff
@@ -319,15 +320,15 @@ const IdvFailed = ({
                     })}
                 >
                     <FormBody className='form-body' scroll_offset={setScrollOffset()}>
-                        <Text size={is_mobile ? 'xs' : 's'} weight='bold' align='center'>
+                        <Text size={isMobile ? 'xs' : 's'} weight='bold' align='center'>
```
Comment (utkarsha-deriv) [inline packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx:323]: In this case, the tablet view will have fontsize of desktop screens. Is this expected?

File: packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx [hunk 6]
```diff
@@ -319,15 +320,15 @@ const IdvFailed = ({
...
                             <Localize i18n_default_text='Your identity verification failed because:' />
                         </Text>
                         {(status?.error_msg || idv_failure?.failure_message) && (
                             <HintBox
                                 className={classNames('proof-of-identity__failed-message', 'hint-box-layout')}
                                 icon='IcAlertDanger'
                                 message={
-                                    <Text as='p' size={is_mobile ? 'xxs' : 'xs'} data-testid={mismatch_status}>
+                                    <Text as='p' size={isMobile ? 'xxs' : 'xs'} data-testid={mismatch_status}>
```
Comment (utkarsha-deriv) [inline packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx:331]: here also

File: packages/account/src/Components/file-uploader-container/file-uploader-container.tsx (hunk 4 above)
Comment (fasihali-deriv) [reply packages/account/src/Components/file-uploader-container/file-uploader-container.tsx:15]: Hey @utkarsha-deriv
As we don't have design for all screens and I checked on tablet view xs is too small for tablet view.

File: packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx (hunk 5 above)
Comment (fasihali-deriv) [reply packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx:323]: Hey @utkarsha-deriv
As we don't have design for all screens and I checked on tablet view xs is too small for tablet view.

File: packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx (hunk 6 above)
Comment (fasihali-deriv) [reply packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx:331]: Hey @utkarsha-deriv
As we don't have design for all screens and I checked on tablet view xs is too small for tablet view.

**Review by amina-deriv — APPROVED** [review]

**Review by amina-deriv — APPROVED** [review]

**Pull request merged**